config.go

```diff
@@ -423,6 +427,7 @@ func NewConfig() *Config {
 	c.Consumer.MaxWaitTime = 250 * time.Millisecond
 	c.Consumer.MaxProcessingTime = 100 * time.Millisecond
 	c.Consumer.Return.Errors = false
+	c.Consumer.Offsets.Enable = true
```

Comment:
Would suggest have a more descriptive name:

```suggestion
	c.Consumer.Offsets.AutoCommitEnable = true
```

Also `c.Consumer.Offsets.AutoCommit.Enable` would be fine.

Reply:
@kjelle can you please make this change?

Reply:
> Also `c.Consumer.Offsets.AutoCommit.Enable` would be fine.

I totally agree @sam-obeid. I am making appropriate changes and reissuing the PR.